source/lib.js

```diff
@@ -832,13 +832,13 @@ function expandChainedComparisons([first, binops]) {
           endIndex = i + 1
         }
 
-        results = results.concat(first, ...binops.slice(start, endIndex))
+        results.push(first, ...binops.slice(start, endIndex).flat())
         first = [exp].concat(binops.slice(index + 1, endIndex))
         start = endIndex
       })
     } else {
       // Advance start if there was no chain
-      results = results.concat(first, ...binops.slice(start, i + 1))
+      results.push(first, ...binops.slice(start, i + 1).flat())
```

Comment:
These changes from `concat` to `push` turned out to not be necessary, but they would be if `first` were even an array (so that it gets treated like a single thing). I left the change in also because it should be faster to be in-place.